resources/assets/lib/beatmap-discussions/nominations.coffee

```diff
@@ -543,7 +543,7 @@ export class Nominations extends React.PureComponent
   renderLoveBeatmapModal: =>
     return if !@state.loveBeatmapModal
 
-    el Modal, visible: true, onClose: @handleLoveBeatmapModal,
+    el Modal, onClose: @handleLoveBeatmapModal,
```

Comment:
not related but it seems odd this one closes when clicking outside the modal but the beatmap owner one doesn't 🤔

Reply:
I think it's to avoid accidentally close the modal. Although it can be fixed by adding prompt, I guess 🤷‍♀️